test/functional/provider/admin/messages/inbox_controller_test.rb

```diff
@@ -48,4 +48,24 @@ def test_index_system_message
     assert_select 'title', "Inbox - Index | Red Hat 3scale API Management"
     assert_select '#export-to-csv', false, 'Export all Messages'
   end
+
+  test "creates valid reply" do
+    login_as(@admin)
+    post :reply, params: { message: { subject: "Valid Message", :body => "message with subject" }, id: @message.id }
```

Comment:
```suggestion
    post :reply, params: { message: { subject: "Valid Message", body: "message with subject" }, id: @message.id }
```
Just a tiny change to use the same syntax as in the rest of the line.